rollup.config.js

```diff
@@ -0,0 +1,29 @@
+import typescript from 'rollup-plugin-typescript2';
+import { terser } from "rollup-plugin-terser";
+import * as path from "path";
+
+const BASE_PATH = process.cwd();
+const getFile = path => path.resolve(BASE_PATH, path);
+
+const pkg = require(getFile('package.json'));
+
+export default {
+  input: path.resolve(BASE_PATH, 'src/index.ts'),
```

Comment:
This seems a strong assumption (although probably always true). No action required, just saying.

Reply:
True, I'm aware of that. On the other hand, there is no other way we could verify that.
`main` field in `package.json` would be my fav, but unfortunately, it usually points to `dist/index.js`.

Reply:
Yes, I was just pointing out; I do not have a real solution for that right now, I think we can leave it as it is.